scheduler.go

```diff
@@ -207,6 +224,7 @@ func (sc *Scheduler) listTasks(ctx context.Context) ([]Task, error) {
 			Schedule:  c.Labels[schedulerLabel],
 			Service:   service,
 			Command:   args,
+			logging:   c.Labels[logsLabel] == "true",
```

Comment:
May I suggest `strconv.ParseBool`?

Reply:
sure thing. go is not my language, so I don't know the tools and libs